timestamp,category,user_id,function,line_number,flag,message,exception
2024-06-08 12:12:29,ERROR,asif24,create_or_pass_folder,197,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 190, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:1e1d598f-601e-0066-106f-b9c75c000000
Time:2024-06-08T06:42:28.6766157Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:1e1d598f-601e-0066-106f-b9c75c000000
Time:2024-06-08T06:42:28.6766157Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-08 12:12:29,ERROR,asif24,create_or_pass_folder,206,0,"Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:1e1d598f-601e-0066-106f-b9c75c000000
Time:2024-06-08T06:42:28.6766157Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:1e1d598f-601e-0066-106f-b9c75c000000
Time:2024-06-08T06:42:28.6766157Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:1e1d598f-601e-0066-106f-b9c75c000000
Time:2024-06-08T06:42:28.6766157Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:1e1d598f-601e-0066-106f-b9c75c000000
Time:2024-06-08T06:42:28.6766157Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 190, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:1e1d598f-601e-0066-106f-b9c75c000000
Time:2024-06-08T06:42:28.6766157Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:1e1d598f-601e-0066-106f-b9c75c000000
Time:2024-06-08T06:42:28.6766157Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-08 12:12:29,INFO,asif24,home,1380,1,User asif24 logged in successfully,
2024-06-08 12:12:30,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 12:12:30,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 12:12:31,INFO,asif24,get_data_source,1984,1,table_update route successfully send data,
2024-06-08 12:12:38,ERROR,asif24,handle_clear_chat,1912,0,clear_chat for ask_question route error,"clear_chat for ask_question route error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 1895, in handle_clear_chat
    Main_Emm_list = Emm['documents']
TypeError: string indices must be integers"
